projects/packages/identity-crisis/src/class-ui.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Identity_Crisis package.
+ *
+ * @package  automattic/jetpack-identity-crisis
+ */
+
+namespace Automattic\Jetpack\IdentityCrisis;
+
+use Automattic\Jetpack\Assets;
+use Automattic\Jetpack\Identity_Crisis;
+use Automattic\Jetpack\Tracking as Tracking;
+use Jetpack_Tracks_Client;
+
+/**
+ * The Identity Crisis UI handling.
+ */
+class UI {
+
+	/**
+	 * Initialization.
+	 */
+	public static function init() {
+		if ( did_action( 'jetpack_identity_crisis_ui_init' ) ) {
+			return;
+		}
+
+		/**
+		 * Action called after initializing Connection UI Admin resources.
+		 *
+		 * @since 1.1.0
+		 */
+		do_action( 'jetpack_identity_crisis_ui_init' );
+
+		$idc_data = Identity_Crisis::check_identity_crisis();
+
+		// TODO: replace the `jetpack_disconnect` check with a non-admin IDC screen.
+		if ( false === $idc_data || ! current_user_can( 'jetpack_disconnect' ) ) {
+			return;
+		}
+
+		$idc_data = Identity_Crisis::check_identity_crisis();
+
+		add_action( 'admin_enqueue_scripts', array( static::class, 'enqueue_scripts' ) );
+
+		Tracking::register_tracks_functions_scripts( true );
+	}
+
+	/**
+	 * Enqueue scripts!
+	 */
+	public static function enqueue_scripts() {
+		if ( is_admin() ) {
+			Assets::register_script(
+				'jp_identity_crisis_banner',
+				'../build/index.js',
+				__FILE__,
+				array(
+					'in_footer'  => true,
+					'textdomain' => 'jetpack',
+				)
+			);
+			Assets::enqueue_script( 'jp_identity_crisis_banner' );
+			wp_add_inline_script( 'jp_identity_crisis_banner', static::get_initial_state(), 'before' );
+			wp_set_script_translations( 'jp_identity_crisis_banner', 'jetpack' );
```

Comment:
This should be unneeded, specifying the textdomain at line 60 should be having `Assets::register_script()` handle it.
```suggestion
```